lib/ffaker/bank.rb

```diff
@@ -92,13 +92,11 @@ def iban(country_code: nil)
     end
 
     def card_number
-      fetch_sample(CARD_NUMBERS)
+      FFaker.numerify('#### #### #### ####')
     end
 
-    def card_expiry_date(**params)
-      params[:year_range] ||= 5
-      params[:year_latest] ||= -params[:year_range] # for the future
-      FFaker::Time.date(params)
+    def card_expiry_date(params = { year_range: 5, year_latest: -5 })
```

Comment:
(maybe) This will cover 'mm/yyyy' form. 
```suggestion
    def card_expiry_date(year_range: 5, year_latest: -5, format: '%m/%y')
      FFaker::Time.date(params).strftime(format)
```

Reply:
done